src/components/pages/Categories/List.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="home">
+    <article class="home__content">
```

Comment:
[imo]

`category` という単語使った方がいいですね

```suggestion
  <div class="categories">
    <article class="categories__content">
```